src/shared/components/form/formEmail/formEmail.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import FormInput from '../formInput/formInput';
+
+class FormEmail extends Component {
+
+  render() {
+    return (
+      <FormInput
+        {...this.props}
+        validationRegex={/\S+@\S+\.\S+/}
```

Comment:
I may have removed the comments when I created the FormInput component, feel free to add them back.